index.js

```diff
@@ -144,6 +144,7 @@ module.exports = {
     "import/no-named-default": "error",
     "import/no-webpack-loader-syntax": "error",
     "import/no-useless-path-segments": ["error", { noUselessIndex: true }],
+    "import/prefer-default-export": "warn",
```

Comment:
This one might be more noise than anything else, let's try.

Reply:
I like this !